src/framegrab/grabber.py

```diff
@@ -559,6 +559,29 @@ def __init__(self, config: dict):
         self.idx = idx
         GenericUSBFrameGrabber.indices_in_use.add(idx)
 
+    def _validate_image_capture(self, capture: cv2.VideoCapture) -> bool:
```

Comment:
At some point we'll want to be able to override this.  People might want to use the B&W cameras.  But for now this is pragmatic.  If it's easy to make this configurable as opt-in for B&W, let's do that.  Otherwise let's put a TODO.